src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -505,6 +505,10 @@ public CoordinateBuilder.Expression getCoordinates() {
         }
 
         public enum Type {
+            Plus,
+            Minus,
+            Div,
+            Mul,
```

Comment:
Our parser doesn't appear to be using these four enum constants anywhere. I will remove them for now, as we already have corresponding constants in `J.Binary.Type` (`Addition`, `Subtraction`, `Multiplication`, and ,`Division`).